util/fhir.ts

```diff
@@ -36,3 +36,26 @@ export function createPatientResourceString(birthDate: string): string {
 export function getPatientInfoString(patient: fhir4.Patient) {
   return `${patient.name?.[0]?.given?.join(' ')} ${patient.name?.[0]?.family} (DOB: ${patient.birthDate})`;
 }
+
+/**
+ * Identifies the valuesets referenced in a DataRequirement and constructs a string which displays
+ * those valuesets
+ * @param dr {Object} a fhir DataRequirement object
+ * @param valueSetsMap {Object} a mapping of valueset urls to valueset names and titles
+ * @returns {String} displaying the valuesets referenced by a DataRequirement
+ */
+export function getDataRequirementFiltersString(dr: fhir4.DataRequirement, valueSetsMap: any): string {
+  const valueSets = dr.codeFilter?.reduce((acc: string[], e) => {
+    if (e.valueSet) {
+      acc.push(valueSetsMap[e.valueSet]);
```

Comment:
I think it might be useful to include the ValueSet OID or full URL in the display, maybe in parentheses? 

```suggestion
      acc.push(`${valueSetsMap[e.valueSet]} (${e.valueSet})`);
```

something like this? Didn't actually test it locally so you don't need to apply the suggestion